src/Symfony/Component/Messenger/DependencyInjection/MessengerPass.php

```diff
@@ -176,12 +183,10 @@ private function registerHandlers(ContainerBuilder $container, array $busIds)
 
         foreach ($busIds as $bus) {
             $container->register($locatorId = $bus.'.messenger.handlers_locator', HandlersLocator::class)
-                ->setArgument(0, $handlersLocatorMappingByBus[$bus] ?? [])
-            ;
+                ->setArgument(0, $handlersLocatorMappingByBus[$bus] ?? []);
```

Comment:
please revert these, they are unrelated

Reply:
This was made by the bot, after the apply patch.